dev/tasks/README.md

```diff
@@ -106,7 +106,7 @@ The script does the following:
    $ git clone https://github.com/kszucs/crossbow
 
    $ cd arrow/dev/tasks
-   $ python crossbow.py
+   $ python crossbow.py submit <task names>
```

Comment:
Is there a way that we can make the `submit` subcommand show a better error message if the task isn't valid?

Right now I get this:

```sh
$ python crossbow.py submit --dry-run linux-packges
Traceback (most recent call last):
  File "crossbow.py", line 409, in <module>
    crossbow(auto_envvar_prefix='CROSSBOW')
  File "/home/phillip/miniconda/envs/pyarrow36/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/phillip/miniconda/envs/pyarrow36/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/phillip/miniconda/envs/pyarrow36/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/phillip/miniconda/envs/pyarrow36/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/phillip/miniconda/envs/pyarrow36/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "crossbow.py", line 338, in submit
    tasks = {name: tasks[name] for name in task_names}
  File "crossbow.py", line 338, in <dictcomp>
    tasks = {name: tasks[name] for name in task_names}
KeyError: 'linux-packges'
```

Reply:
Addressed in https://github.com/kszucs/arrow/pull/3